docs/k0sctl-install.md

```diff
@@ -1,54 +1,67 @@
-# Installing with k0sctl
+# Install using k0sctl
 
-This tutorial is based on k0sctl tool and it's targeted for creating a multi-node cluster for remote hosts. It describes an install method, which is automatic and easily repeatable. This is recommended for production clusters and the automatic upgrade requires using this install method. The automatic upgrade process is also described in this tutorial.
-
-k0sctl is a command-line tool for bootstrapping and managing k0s clusters. k0sctl connects to the provided hosts using SSH and gathers information about the hosts. Based on the findings it proceeds to configure the hosts, deploys k0s and connects the k0s nodes together to form a cluster.
+k0sctl is a command-line tool for bootstrapping and managing k0s clusters. k0sctl connects to the provided hosts using SSH and gathers information on
+the hosts, with which it forms a cluster by configuring the hosts, deploying k0s, and then connecting the k0s nodes together.
 
 ![k0sctl deployment](img/k0sctl_deployment.png)
 
-### Prerequisites
+With k0sctl, you can create multi-node clusters for remote hosts in a manner that is automatic and easily repeatable. This method is recommended for production cluster installation.
 
-k0sctl can be executed on Linux, MacOS and Windows. See more details from the [k0sctl github repository](https://github.com/k0sproject/k0sctl). For hosts running k0s, see the [System Requirements](system-requirements.md).
+**Note**: The k0sctl install method is necessary for automatic upgrade.
```

Comment:
Not sure where this came from or what it means. There is no automatic upgrade AFAIK. cc: @jnummelin